core/embed/rust/src/ui/macros.rs

```diff
@@ -4,3 +4,35 @@ macro_rules! include_res {
         include_bytes!(concat!(env!("CARGO_MANIFEST_DIR"), "/src/ui/", $filename))
     };
 }
+
+#[allow(unused_macros)]
+/// Concatenates arbitrary amount of slices into a String.
+macro_rules! build_string {
+    ($max:expr, $($string:expr),+) => {
+        {
+            let mut new_string = String::<$max>::new();
+            $(new_string.push_str($string).unwrap();)+
+            new_string
+        }
+    }
+}
+
+#[allow(unused_macros)] // Mostly for debugging purposes.
+/// Transforms integer into string slice. For example for printing.
+macro_rules! inttostr {
+    ($int:expr) => {{
+        heapless::String::<10>::from($int).as_str()
+    }};
+}
+
+#[allow(unused_macros)] // Mostly for debugging purposes.
+/// Transforms bool into string slice. For example for printing.
+macro_rules! booltostr {
+    ($bool:expr) => {{
+        if $bool {
+            "true"
+        } else {
+            "false"
+        }
+    }};
+}
```

Comment:
This can be just a regular function as opposed to macro, or am I missing something?